### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
VSTS stuff assigned to me

**407002** Accessibility : MAS05A : Keyboard R Tools+R Packages : KB focus invisible over package name in package description pane
**402127** Accessibility : MAS40B : Inspect : Name property is empty for all Edit boxes and combo boxes in "Import DataSet" window
**407002** Accessibility : MAS05A : Keyboard R Tools+R Packages : KB focus invisible over package name in package description pane
**406548** Accessibility : MAS40B : INPSECT : R Tools + R Packages : Name property for the thumb is null in R Packages window
**406112** Accessibility : MAS40B : INPSECT : R Tools + R Packages : Name property for buttons is null in R Packages window
**406109** Accessibility : MAS40 : INSPECT : R Tools + R Plot History : Device Number and expand collapse button in R Plot History is not programatically defined
**406110** Accessibility : MAS40B : INSPECT : R Tools + Workspace : Name property for the edit in Workspace window is null

### CODE REVIEW

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml [hunk 1]
```diff
@@ -140,7 +140,8 @@
                                  Background="{DynamicResource {x:Static rwpf:Brushes.BackgroundBrushKey}}" 
                                  Foreground="{DynamicResource {x:Static rwpf:Brushes.UITextKey}}"
                                  ToolTip="{Binding Path=NameTextBoxTooltip}"
-                                 IsVisibleChanged="TextBoxName_IsVisibleChanged"/>
+                                 IsVisibleChanged="TextBoxName_IsVisibleChanged"
+                                 AutomationProperties.Name="{Binding Path=NameTextBoxTooltip}"/>
```
Comment (AlexanderSher) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: I'm not sure that we need a `Name` automation property for the text box. What bug is this?

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml [hunk 2]
```diff
@@ -167,7 +170,8 @@
                                     VerticalAlignment="Center" Click="ButtonSave_Click" 
                                     Style="{StaticResource ToolWindowButtonStyle}"  ToolTipService.ShowOnDisabled="True"
                                     Content="{x:Static components:Resources.ConnectionManager_Save}" 
-                                    ToolTip="{Binding Path=SaveButtonTooltip}">
+                                    ToolTip="{Binding Path=SaveButtonTooltip}"
+                                    AutomationProperties.Name="{Binding Path=SaveButtonTooltip}">
```
Comment (AlexanderSher) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: You don't need to specify automation name for button if its content is string:
https://referencesource.microsoft.com/#PresentationFramework/Framework/System/Windows/Automation/Peers/ButtonBaseAutomationPeer.cs,5916f54bbfafd784,references

Comment (MikhailArkhipov) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: Yeah except bugs are about how Inspect sees it and if control doesnt have name, but is opened.

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml (hunk 1 above)
Comment (MikhailArkhipov) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: 406110

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml (hunk 2 above)
Comment (AlexanderSher) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: Inspect sees what is expected:

image

What do you mean by "control is opened"?

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml [hunk 3]
```diff
@@ -284,12 +289,14 @@
                         </Button>
 
                         <!-- edit/properties button -->
-                        <Button x:Name="ButtonEdit" Grid.Row="0" Grid.Column="4" Style="{StaticResource EditButtonStyle}">
+                        <Button x:Name="ButtonEdit" Grid.Row="0" Grid.Column="4" Style="{StaticResource EditButtonStyle}"
+                                AutomationProperties.Name="{x:Static components:Resources.ConnectionManager_EditConnection}">
```
Comment (AlexanderSher) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: These buttons (`ButtonEdit` and `ButtonDelete`) have automation name specified in their styles already

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml (hunk 2 above)
Comment (MikhailArkhipov) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: No idea, its Wipro. Simpler to provide it than to argue, they go by checklist.

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml (hunk 3 above)
Comment (MikhailArkhipov) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: Ah, OK, missed that

File: src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml (hunk 2 above)
Comment (AlexanderSher) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: Ok

Comment (AlexanderSher) [inline src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml]: Ok

**Comment by AlexanderSher** [comment]
👍 , please remove `AutomationProperties.Name` from `ButtonEdit` and `ButtonDelete`

**Pull request merged**